src/chatlog/chatmessage.cpp

```diff
@@ -175,6 +181,72 @@ void ChatMessage::hideDate()
         c->hide();
 }
 
+QString ChatMessage::detectMarkdown(const QString &str)
+{
+    QString out;
+
+    // Create regex for certain markdown syntax
+    QRegExp exp("(\\*\\*)([^\\*\\*]{2,})(\\*\\*)"   // Bold    **text**
+                "|(\\*)([^\\*]{2,})(\\*)"           // Bold *text*
+                "|(\\_)([^\\_]{2,})(\\_)"           // Italics _text_
+                "|(\\_\\_)([^\\_\\_]{2,})(\\_\\_)"  // Italics __text__
```

Comment:
Yeah, then I had someone else tell me having ** and * for bold would make more sense, rather than having three Italics rules.

Though it would balance if we switched the __'s which we hadn't considered. 

I was just going off [this](https://github.com/tux3/qTox/pull/2832#issuecomment-174231662) lol.